experimental/Schemes/CoveredScheme.jl

```diff
@@ -278,6 +278,8 @@ end
     R, x = PolynomialRing(kk, [Symbol("("*String(s[k+1])*"//"*String(s[i+1])*")") for k in 0:r if k != i])
     phi = hom(S, R, vcat(gens(R)[1:i], [one(R)], gens(R)[i+1:r]))
     I = ideal(R, phi.(gens(defining_ideal(X))))
+    # TODO: Temporary fix because ideal membership is broken in MPolyQuos.
+    #push!(U, Spec(MPolyQuoLocalizedRing(R, I, units_of(R))))
```

Comment:
Is there an issue tracking this?

Reply:
No, this is deprecated. I forgot to remove it.